src/main/antlr/StitchingDSL.g4

```diff
@@ -61,6 +61,8 @@ fieldArgumentReference : '$argument' '.' name ;
 
 contextArgumentReference : '$context' '.' name ;
 
+indexReference: '$index' ;
```

Comment:
not sure about this. we could add a completely different thing like `object indexed` instead of `object identified by` @bbakerman thoughts?

Reply:
Ah yeah, that would work better. I'm happy to change that.